src/mqt/bench/benchmarks/random.py

```diff
@@ -0,0 +1,22 @@
+from mqt.bench import utils
+from qiskit import QuantumCircuit, transpile
+from qiskit.circuit.random import random_circuit
+
+
+def create_circuit(num_qubits: int) -> QuantumCircuit:
+    """Returns a random quantum circuit twice as deep as wide. The random gate span over four qubits maximum.
+
+    Keyword arguments:
+    num_qubits -- number of qubits of the returned quantum circuit
+    """
+    qc = random_circuit(num_qubits, num_qubits * 2, measure=False, seed=10)
+    gates = list(set(utils.get_openqasm_gates()) - {"rccx", "csx", "cu"})
+    qc = transpile(
+        qc,
+        basis_gates=gates,
+        seed_transpiler=10,
+        optimization_level=1,
+    )
+    qc.measure_all()
```

Comment:
```suggestion
    qc = random_circuit(num_qubits, num_qubits * 2, measure=True, seed=10)
```
Why not just?

Reply:
There are two reasons for that. 
1) The `measure=True` does not add a barrier in front of the measurements. Could be neglected, but all other benchmark circuits have such a barrier and I wanted it to be consistent.
2) The transformation from a `Qiskit.QuantumCircuit` to a `pytket.Circuit` via the `pytket-qiskit` transformation functions crashes if those mentioned gates are used why I removed them using that transpilation pass. I am aware that this is not ideal.

Reply:
I get the consistency argument. +1 on that!

The issue with the gates is non-ideal. Would you mind creating an issue in the tket repository for that?

Reply:
Got point, I created the issue [here](https://github.com/CQCL/pytket-qiskit/issues/138).